app/views/school_groups/recent_usage.html.erb

```diff
@@ -1 +1,49 @@
 <%= render 'enhanced_header' %>
+
+<table width="100%" class="mt-3 table table-borderless table-sorted school-groups-recent-usage-table">
+  <thead>
+    <tr>
+      <th colspan="2"><%= t('common.electricity') %></th>
```

Comment:
Can you try adding the fuel type icons, as we have on the table on the school dashboard?